wagtailio/static/sass/abstracts/_variables.scss

```diff
@@ -9,6 +9,7 @@ $color--eggshell: #B3E7E8;
 $color--light-grey: #EDEAF3;
 $color--white:#FFF;
 $color--pink: #F9CBCE;
+$color--off-black-30: rgb(38 38 38 / 30%);
```

Comment:
Could we use hex for this, please? I think this is `#4d4d4d`

Reply:
This is a semi-transparent colour so it needs the alpha channel value. The equivalent hex value is `#2626264d` so I can update it to that 👍